src/plugins/intel_cpu/src/transformations/snippets/x64/pass/eliminate_brgemm_copy_b.cpp

```diff
@@ -30,12 +31,28 @@ pass::EliminateBrgemmCopyB::EliminateBrgemmCopyB() {
 
         const auto& in_desc = snippets::lowered::PortDescriptorUtils::get_port_descriptor_ptr(copy_b_node->input(0));
         const auto& layout = in_desc->get_layout();
-        // TODO:
-        // 1. Ticket 157340: support external repacking for copyB with compensations
-        // 2. Ticket 157339: support external repacking for non-planar layout
-        if (!ov::snippets::utils::is_planar_layout(layout) ||
-            brgemm_utils::with_compensations(copy_b_node->get_type()) || transformation_callback(copy_b_node))
+
+        auto is_supported_layout = [](const std::vector<size_t>& layout) {
+            return layout.empty() || (layout.size() - 1 == layout.back());
```

Comment:
Yeah, I added propagation to config. But left `is_supported_layout` as it is

Reply:
Found that in `transposed_b=true` case we should [allocate more memory](https://github.com/openvinotoolkit/openvino/blob/45bf77b842077a058e60f1de47626397c1d1e8e8/src/plugins/intel_cpu/src/transformations/snippets/x64/pass/lowered/expressions/brgemm_copy_b_buffer_expressions.cpp#L49). Because of that, I decided to leave it so for now.